packages/api/portfolioDrafts/submit/submitPortfolioDraft.ts

```diff
@@ -33,6 +36,12 @@ export async function handler(event: APIGatewayProxyEvent): Promise<APIGatewayPr
 
   try {
     const result = await submitPortfolioDraftCommand(TABLE_NAME, portfolioDraftId);
+    const data = await sqsClient.send(
+      new SendMessageCommand({
+        QueueUrl: QUEUE_URL,
+        MessageBody: JSON.stringify(result.Attributes as PortfolioDraft),
```

Comment:
We may just want to have this be the Portfolio Draft ID and then let the consuming Function (in the Orchestrator) look it up. What is the value of `Attributes` here? If it contains the full value of the Item being updated then we can keep it as-is, but otherwise let's avoid doing the extra lookup here.

Reply:
`result.Attributes` is the PortfolioDraft object that is submitted, it doesn't require an extra look up. Right now the subscribe fn just logs to PortfolioDraft : 
```
{
    "submit_id": "5db2ad5d-7214-4a6e-b366-b3bdbd81f8d1",
    "updated_at": "2021-10-01T22:35:07.504Z",
    "created_at": "2021-10-01T22:34:49.189Z",
    "num_applications": 0,
    "num_portfolio_managers": 0,
    "status": "in_progress",
    "num_environments": 0,
    "num_task_orders": 0,
    "id": "8c90414c-a367-40e2-90b5-4510dc72292d",
    "name": ""
}
```

Reply:
OK cool, let's keep it as-is then.